### DISCUSSION TITLE
Toaster styles lost?

### DISCUSSION THREAD

**Discussion by tblackwell-tm** [epicweb-dev/epic-stack#502] [Q&A]
Hi, I've adopted the Epic Stack's approach for toast messages but I've run into a problem. When a user triggers an error boundary the Toaster's styles are ejected from the document head. One workaround would be download the current version of the CSS and import that into the component, but obviously that isn't ideal.

Is there a fix in the Epic Stack for this already and I've missed it? If not, can anyone think of a decent way of solving the issue?

Thanks
Tom

**Comment by kentcdodds** [comment]
The top level `ErrorBoundary` (in the root.tsx) is a backup error boundary and can't reliably have access to the loader data which is why this is happening.

You'll want to make sure to have more error boundaries throughout the routes so you can avoid errors bubbling up to the top.
